sdk/eventhub/Azure.Messaging.EventHubs/src/Core/TransportEventHubConsumer.cs

```diff
@@ -26,7 +26,7 @@ internal abstract class TransportEventHubConsumer
         public abstract void UpdateRetryPolicy(EventHubRetryPolicy newRetryPolicy);
 
         /// <summary>
-        ///   Receives a bach of <see cref="EventData" /> from the the Event Hub partition.
+        ///   Receives a batch of <see cref="EventData" /> from the Event Hub partition.
```

Comment:
I was sad it was only receiving Bach, what about Vivaldi?! 😢

Reply:
Behind Handel and Tchaikovsky on the priority list.